<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Users</class>
 <widget class="QDialog" name="Users">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель адміністратора</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1,0">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Дод&amp;ати</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Редаг&amp;увати</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Видалити</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblInfo">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Закр&amp;ити вікно</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
